tests/ondemand/ondemand_json_path_tests.cpp

```diff
@@ -315,7 +315,7 @@ namespace json_path_tests {
         std::vector<car_type> content;
         for (int i = 0; i < 3; i++) {
             ondemand::object obj;
```

Comment:
That is fine (`std::string("[")`) but a more idiomatic way might be `"["s` instead.

See https://lemire.me/blog/2023/07/05/having-fun-with-string-literal-suffixes-in-c/

(This is not a request for change.)

Reply:
(Though it is not a request for change, it could make the examples nicer.)

Reply:
(String suffixes are C++14 and we want the core library to compile with C++11, so we can't use string suffixes in simdjson itself, but for tests and documentation, that's fine.)

Reply:
Sure, I can change it when #2128 is merged.

Tried changing it now and got some build errors.